_data/sidebars/marketplace_dev_sidebar.yml

```
@@ -418,6 +418,13 @@ entries:
         - title: UI components library
           nested:
 
+            - title: "Actions"
+              url: /docs/marketplace/dev/front-end/ui-components-library/actions/
+              nested:
+
+                - title: "Actions Notification"
```

Please apply it for all sub `Actions` menu items to avoid duplication.
```suggestion
                - title: "Notification"
```